app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  before_save :update_likes_count
+
+  def update_likes_count
+    post = Post.find_by_id(post_id)
+    post.likes_counter = post.likes_counter.to_i + 1
+    post.save
+  end
```

Comment:
- [ ] The same suggestions given to post model, apply to this as well
- Make the method private since it's being called in the callback
- It would be a great idea to use `after_save` callback

Reply:
@descholar-ceo Sure thing to do, thank you so much for your suggestions. And you are right, it is always better to on a safe side 😎